packages/web/app/src/lib/supertokens/third-party-email-password-node-oidc-provider.ts

```diff
@@ -1,8 +1,8 @@
-import type { InternalApi } from '@hive/server';
-import type { inferRouterProxyClient } from '@trpc/client';
+import { InternalApi } from '@hive/server';
+import { inferRouterProxyClient } from '@trpc/client';
 import { ExpressRequest } from 'supertokens-node/lib/build/framework/express/framework';
 import ThirdPartyEmailPasswordNode from 'supertokens-node/recipe/thirdpartyemailpassword';
-import type { TypeInput as ThirdPartEmailPasswordTypeInput } from 'supertokens-node/recipe/thirdpartyemailpassword/types';
+import { TypeInput as ThirdPartEmailPasswordTypeInput } from 'supertokens-node/recipe/thirdpartyemailpassword/types';
```

Comment:
@B2o5T here it changed from `type` import to regular import. Is that valid?

Reply:
Yeah it’s doesn't matter